### ISSUE TITLE
This file does not contain any Objective-C runtime information.

### ISSUE THREAD

**Issue by wushiwein119** [abynim/Sketch-Headers#26]
when i run "class-dump -H sketchtool -o /output" , show error "This file does not contain any Objective-C runtime information." . please help me

**Comment by abynim** [comment]
Yeah it seems like sketchtool is a different kind of executable and class-dump is unable to read it. You might need to use a [disassembler](https://www.hopperapp.com) to peek at the headers instead of using class-dump. 

I'm curious why you're looking for sketchtool headers though.

**Comment by wushiwein119** [comment]
> Yeah it seems like sketchtool is a different kind of executable and class-dump is unable to read it. You might need to use a [disassembler](https://www.hopperapp.com) to peek at the headers instead of using class-dump.
> 
> I'm curious why you're looking for sketchtool headers though.

sorry, my mistake, I use "class-dump -H Sketch -o /output"
